02-intermediate-front-end/scope-context-closures/closures.md

```diff
@@ -181,3 +183,5 @@ Note, the interface to the object 'bob' is two functions/methods, getName and ge
 The implementation are the private variables `_name`, `_age` and the code inside the getName and getAge functions.
 
 Client code does **NOT** need to be concerned with implementation details. Only with the interface.
+
+> I don't see IIFEs mentioned anywhere... Were you going to include that?
```

Comment:
@RobertAKARobin Good call! I'll add a short section near the end, but tag it as optional since this lesson went right to time last time.